File: Modul 7/Soal 3/PRAK603-2210817220007-AISYA ANINDIA SEPTHA.c
#include <stdio.h>
int main()
{
int a, b, c, d, e;
scanf("%d %d", &a, &b);
int f[a], g[b];
    if(a != b)
        {printf("\nJumlah tidak sama");}
    else
        {for(e=0; e<a; e++)
            {scanf("%d", &c);
            f[e] = c;
            }
        for(e=0; e<b; e++)
            {scanf("%d", &c);
            g[e] = c;
            }
        printf("\n");
        for(e=0; e<b; e++)
            {d = f[e] * g[e];
            printf("%d ", d);
            }
        }
printf("\n");
return 0;
}

File: Modul 7/Soal 3/PRAK603-2210817220007-AISYA ANINDIA SEPTHA.py
a, b = input().split()
a = int (a)
b = int (b)
if (a != b) :
    print("Jumlah tidak sama")
    print()
else :
    c = list (map (int, input().split()))
    d = list (map (int, input().split()))
    print("")
    for e in range (a) :
        f = c[e] * d[e]
        print(f, end=' ')
    print("\n")